src/tron/TronEngine.ts

```diff
@@ -481,6 +482,15 @@ export class TronEngine extends CurrencyEngine<TronTools, SafeTronWalletInfo> {
 
     const ourReceiveAddresses: string[] = []
 
+    const memos: EdgeMemo[] = []
+    if (data != null) {
+      memos.push({
+        type: 'text',
+        value: Buffer.from(base16.parse(data)).toString('ascii'),
```

Comment:
Are memos really ASCII? I would have expected UTF-8, at least. What if the raw bytes aren't valid UTF-8? Do we need a try/catch around this? Would it make sense for the `catch` block to emit a "hex" memo with the raw bytes that we can't decode?